grafana/scylla-dash.master.template.json

```diff
@@ -229,7 +231,8 @@
                                 "step": 10
                             }
                         ],
-                        "title": "Foreground Writes"
+                        "title": "Foreground Writes",
+                        "description": "Foreground writes are writes that weren't acknowledged yet to the application. For instance, if a single replica responded and two are needed. This metric represents a queue size, not a rate. High values here correlate with increased write latencies."
```

Comment:
A suggestion (you can ignore if you want to)
if a single replica responded but two are needed (because of the consistency level)